server/src/models/Signup.js

```diff
@@ -0,0 +1,26 @@
+const Model = require("./Model")
+
+class Signup extends Model {
+  static get tableName() {
+    return "signups"
+  }
+
+  static get jsonSchema() {
+    return {
+      type: "object",
+      required: ["userId", "planId"],
+      properties: {
+        userId: { type: ["integer", "string"] },
+        planId: { type: ["integer", "string"] }
+      }
+    }
+  }
+
+  static get relationMappings() {
+    return {
+
+    }
+  }
```

Comment:
if you dont have a relationmapping yet, then I would remove this till you are ready